pkg/namesgenerator/names-generator.go

```diff
@@ -358,6 +358,9 @@ var (
 	rnd = rand.New(random.NewSource())
 )
 
+// GetRandomName generates a random name from the list of adjectives and surnames in this package
+// formatted as "adjective_surname". For example 'focused_turing'i. If retry is non-zero, a random
```

Comment:
@LK4D4 nope - https://github.com/dave-tucker/docker/blob/golint_namesgen/pkg/namesgenerator/names-generator.go#L34

according to google it can be `focused` or `focussed` - I would have used the later but meh. yay for ambiguous english!